### ISSUE TITLE
Linaria crashes webpack/rollup

### ISSUE THREAD

**Issue by nstepien** [callstack/linaria#1062]
## Environment

- Linaria version: 4.2.0
- Bundler (+ version): webpack@5.74.0, rollup@3.1.0
- Node.js version: 18.8.0
- OS: Win11

## Description

```
webpack 5.74.0 compiled with 1 error in 1118 ms
ERROR in ./src/Component.tsx
Module build failed (from ./node_modules/@linaria/webpack5-loader/lib/index.js):
TypeError: D:\repos\repo\src\Component.tsx: Property name of JSXOpeningElement expected node to be of a type ["JSXIdentifier","JSXMemberExpression","JSXNamespacedName"] but instead got undefined
    at Object.validate (D:\repos\repo\node_modules\@babel\types\lib\definitions\utils.js:131:11)   
    at validateField (D:\repos\repo\node_modules\@babel\types\lib\validators\validate.js:24:9)     
    at validate (D:\repos\repo\node_modules\@babel\types\lib\validators\validate.js:17:3)
    at NodePath._replaceWith (D:\repos\repo\node_modules\@babel\traverse\lib\path\replacement.js:169:5)
    at NodePath._remove (D:\repos\repo\node_modules\@babel\traverse\lib\path\removal.js:59:10)     
    at NodePath.remove (D:\repos\repo\node_modules\@babel\traverse\lib\path\removal.js:38:8)       
    at D:\repos\repo\node_modules\@linaria\utils\lib\scopeHelpers.js:306:42
    at mutate (D:\repos\repo\node_modules\@linaria\utils\lib\scopeHelpers.js:337:19)
    at D:\repos\repo\node_modules\@linaria\utils\lib\scopeHelpers.js:305:5
    at Array.forEach (<anonymous>)
```

## Reproducible Demo

The file where babel/webpack crashes doesn't import linaria, nor does it import any linaria class names 🤷‍♂️ 
Let me know if you still want a reproducible demo

**Comment by nstepien** [comment]
Also having the same crash in rollup
```
[!] (plugin linaria) TypeError: my-repo\src\components\Overlay\Overlay.tsx: Property name of JSXOpeningElement expected node to be of a type ["JSXIdentifier","JSXMemberExpression","JSXNamespacedName"] but instead got undefined
src/components/Overlay/Overlay.tsx
TypeError: my-repo\src\components\Overlay\Overlay.tsx: Property name of JSXOpeningElement expected node to be of a type ["JSXIdentifier","JSXMemberExpression","JSXNamespacedName"] but instead got undefined
    at Object.validate (my-repo\node_modules\@babel\types\src\definitions\utils.ts:138:11)
    at validate (my-repo\node_modules\@babel\types\src\validators\validate.ts:32:9)
    at validateField (my-repo\node_modules\@babel\types\src\validators\validate.ts:19:3)
    at NodePath.validate [as _replaceWith] (my-repo\node_modules\@babel\traverse\src\path\replacement.ts:209:5)
    at NodePath._replaceWith [as _remove] (my-repo\node_modules\@babel\traverse\src\path\removal.ts:42:10)    
    at NodePath._remove [as remove] (my-repo\node_modules\@babel\traverse\src\path\removal.ts:22:8)
    at remove (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:336:28)
    at fn (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:365:19)
    at mutate (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:335:5)
    at Array.forEach (<anonymous>)
    at forEach (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:322:9)
    at removeWithRelated (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:341:5)
    at removeWithRelated (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:422:3)
    at mutate (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:102:5)
    at findParentForDelete (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:145:12)
    at findParentForDelete (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:241:14)
    at findParentForDelete (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:323:26)
    at Array.forEach (<anonymous>)
    at forEach (my-repo\node_modules\@linaria\utils\src\scopeHelpers.ts:322:9)
    at Identifier (my-repo\node_modules\@linaria\babel-preset\src\plugins\preeval.ts:103:13)
    at NodePath.call [as _call] (my-repo\node_modules\@babel\traverse\src\path\context.ts:34:20)
    at NodePath._call [as call] (my-repo\node_modules\@babel\traverse\src\path\context.ts:19:17)
    at NodePath.call [as visit] (my-repo\node_modules\@babel\traverse\src\path\context.ts:92:31)
    at TraversalContext.visit [as visitQueue] (my-repo\node_modules\@babel\traverse\src\context.ts:144:16)    
    at TraversalContext.visitQueue [as visitSingle] (my-repo\node_modules\@babel\traverse\src\context.ts:108:19)
    at TraversalContext.visitSingle [as visit] (my-repo\node_modules\@babel\traverse\src\context.ts:176:19)   
    at visit (my-repo\node_modules\@babel\traverse\src\traverse-node.ts:34:17)
    at NodePath.visit (my-repo\node_modules\@babel\traverse\src\path\context.ts:99:21)
    at TraversalContext.visit [as visitQueue] (my-repo\node_modules\@babel\traverse\src\context.ts:144:16)    
    at TraversalContext.visitQueue [as visitMultiple] (my-repo\node_modules\@babel\traverse\src\context.ts:98:17)
    at TraversalContext.visitMultiple [as visit] (my-repo\node_modules\@babel\traverse\src\context.ts:174:19) 
    at visit (my-repo\node_modules\@babel\traverse\src\traverse-node.ts:34:17)
    at NodePath.visit (my-repo\node_modules\@babel\traverse\src\path\context.ts:99:21)
    at TraversalContext.visit [as visitQueue] (my-repo\node_modules\@babel\traverse\src\context.ts:144:16)    
    at TraversalContext.visitQueue [as visitSingle] (my-repo\node_modules\@babel\traverse\src\context.ts:108:19)
    at TraversalContext.visitSingle [as visit] (my-repo\node_modules\@babel\traverse\src\context.ts:176:19)   
    at visit (my-repo\node_modules\@babel\traverse\src\traverse-node.ts:34:17)
    at NodePath.visit (my-repo\node_modules\@babel\traverse\src\path\context.ts:99:21)
    at TraversalContext.visit [as visitQueue] (my-repo\node_modules\@babel\traverse\src\context.ts:144:16)    
    at TraversalContext.visitQueue [as visitSingle] (my-repo\node_modules\@babel\traverse\src\context.ts:108:19)
    at TraversalContext.visitSingle [as visit] (my-repo\node_modules\@babel\traverse\src\context.ts:176:19)   
    at visit (my-repo\node_modules\@babel\traverse\src\traverse-node.ts:34:17)
    at NodePath.visit (my-repo\node_modules\@babel\traverse\src\path\context.ts:99:21)
    at TraversalContext.visit [as visitQueue] (my-repo\node_modules\@babel\traverse\src\context.ts:144:16)    
    at TraversalContext.visitQueue [as visitMultiple] (my-repo\node_modules\@babel\traverse\src\context.ts:98:17)
    at TraversalContext.visitMultiple [as visit] (my-repo\node_modules\@babel\traverse\src\context.ts:174:19) 
    at visit (my-repo\node_modules\@babel\traverse\src\traverse-node.ts:34:17)
    at NodePath.visit (my-repo\node_modules\@babel\traverse\src\path\context.ts:99:21)
    at TraversalContext.visit [as visitQueue] (my-repo\node_modules\@babel\traverse\src\context.ts:144:16)    
    at TraversalContext.visitQueue [as visitSingle] (my-repo\node_modules\@babel\traverse\src\context.ts:108:19)
```

**Comment by nstepien** [comment]
@Anber FWIW we had to pin linaria to these last-known working versions:
```json
    "@linaria/babel-preset": "4.1.2",
    "@linaria/core": "4.1.1",
    "@linaria/logger": "4.0.0",
    "@linaria/rollup": "4.1.2",
    "@linaria/shaker": "4.1.2",
    "@linaria/tags": "4.1.1",
    "@linaria/utils": "4.1.1",
    "@linaria/webpack5-loader": "4.1.2",
```

**Comment by nstepien** [comment]
Still happening with yesterday's release (`@linaria/core@4.2.0`).

**Comment by Anber** [comment]
Hi @nstepien!

Fixed in #1090 and will be released soon.

**Comment by nstepien** [comment]
Thanks a lot!
We'll be able to upgrade all our repos to the latest version of Linaria now! 🎉
